### PR TITLE
[fix](multi-catalog)The external catalog does not have verification parameters, causing an error to be reported when creating the catalog.

### LINKED ISSUE #24345: [Bug] The external catalog does not have verification parameters, causing an error to be reported when creating the catalog.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

The external catalog does not have verification parameters, causing an error to be reported when creating the catalog.

### What You Expected?

Check the parameter, and prompts the which parameter setting is unreasonable

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liugddx**
## Proposed changes

Issue Number: close #24345

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liugddx** [comment]
run buildall

**Review by qidaye — DISMISSED** [review]
LGTM

**Review by qidaye — DISMISSED** [review]
LGTM

**Comment by qidaye** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalCatalog.java [hunk 1]
```diff
@@ -44,6 +45,9 @@ public class EsExternalCatalog extends ExternalCatalog {
 
     private static final Logger LOG = LogManager.getLogger(EsExternalCatalog.class);
     private EsRestClient esRestClient;
+    private static final List<String> REQUIRED_PROPERTIES = Lists.newArrayList(
+            EsResource.HOSTS
+    );
```
Comment (BePPPower) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalCatalog.java:50]: Maybe it is better to use `ImmutableList` or `ImmutableSet`？The same goes for other `REQUIRED PROPERTIES`.

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by Jibing-Li — APPROVED** [review]

**Pull request merged**